packit_service/constants.py

```diff
@@ -58,6 +58,13 @@
 
 CELERY_DEFAULT_MAIN_TASK_NAME = "task.steve_jobs.process_message"
 
+MSG_MORE_DETAILS = "You can find more details about the job [here]({url}).\n\n"
```

Comment:
I am thinking about how we can make the URL more visible.

Maybe put it on a table?

| | |
| --- | --- |
| job details | https://dashboard.packit.dev/results/copr-builds/5000 |

Or some icon/button/image with a link?

But that's really a small thing we can easily improve later.

Reply:
I don't like the table without a header so much but will try to find some better and more visible way. I will leave it simple as it is for now.